### DISCUSSION TITLE
Made with Webmidi Apps?

### DISCUSSION THREAD

**Discussion by intelliriffer** [djipco/webmidi#161] [Show and tell]
Hi,
Is there some listing of projects done with webmidi.js? Just curious what others are building.
Here are few of my personal projects, any of them needing midi is using WebMidi.js :  https://midi.amitszone.com/
Thx

**Comment by djipco** [comment]
Thanks for sharing!

I'm actually working on precisely such a list for the new website. Meanwhile, you can check out GitHub projects that are [dependent on WebMidi.js](https://github.com/djipco/webmidi/network/dependents) .

**Reply by intelliriffer** [reply]
Thx for the Github project link. :)

**Comment by edellaq** [comment]
I'm currently working on a SPA for guitar fretboard visualisation, as a guinea pig for both Webmidi and the new VUE.JS 3 Composition API... (struggling bit, right now :) hope to have something working sooner or later.
But... what about the new web midi version 3? will it be compatible with the old 2.* code?
What will be the main new features/differences?

**Comment by djipco** [comment]
> I'm currently working on a SPA for guitar fretboard visualisation

Awesome! When it's done, don't forget to share!

> But... what about the new web midi version 3? 

To know more about v3, you can check out the [README](https://github.com/djipco/webmidi/tree/develop#readme) in the develop branch. You can also review the new [documentation for v3](https://webmidijs.org/docs/) and my [todo list](https://github.com/djipco/webmidi/wiki#enhancements-for-upcoming-v3). 

> will it be compatible with the old 2.* code?

I'm working very hard to make it backwards-compatible. Hopefully, migration will be easy. I will know more in a month or two. Actually, I will probably be asking current users to try out the new version in their project to help me troubleshoot backwards-compatibility.
